bigbluebutton-html5/public/locales/en.json

```diff
@@ -108,6 +108,12 @@
     "app.timer.hours": "hours",
     "app.timer.minutes": "minutes",
     "app.timer.seconds": "seconds",
+    "app.timer.abbreviation.hours": "h",
```

Comment:
I don't Believe that abbreviation is a good way to handle it, probably it doesn't make sense in some supported languages, I suggest you or create topics include the full word